### DISCUSSION TITLE
removes the tailwindcss classes when optimizing

### DISCUSSION THREAD

**Discussion by lewandowskit** [tailwindlabs/tailwindcss#4642] [Help]
hello, I'm learning tailwind and I have a problem, how do I set up code optimization for production, it removes responsiveness from the grid framework, what I'm doing wrong, you can guide me

> module.exports = {
  purge: {
    enabled: true,
    content: [
        './**/*.html'
    ]
},
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
      fontFamily: {
        'sans': ['Fira Sans Condensed'],
        'serif': ['Fira Sans Condensed'],
        'mono': ['Fira Sans Condensed'],
        'display': ['Fira Sans Condensed'],
        'body': ['Fira Sans Condensed'],
       },
       colors: {
        primaryorange: '#eb891b',
        secondary: '#1da838',
        lightyellow: '#FBBC13',
        darkyellow: '#FFA100',
        myblue:  '#074697',
        mydarkblue: '#00112D',
      },
    },
  },
  variants: {
  extend: {},
  },
  plugins: [
    require('tailwindcss'),
    require('autoprefixer'),
  ],
}; <

**Comment by VicGUTT** [comment]
Hey !

First and foremost and for future reference, when pasting code, please ensure the formatting is correct and readable. You can do so in markdown by writing it like so  :
\`\`\`js
    // my code JS goes here
\`\`\`

\`\`\`html
    // my code HTML goes here
\`\`\`

\`\`\`css
    // my code CSS goes here
\`\`\`

\`\`\`php
    // my code PHP goes here
\`\`\`

etc...

And for the issue at hand, am I right to assume you're using Tailwind V2 ? Are you sure the purge section is correctly configured (point to the right path) ? Could we get a sample of your template where the issue is happening ? How is your build tool configured ? What build tool are you using (Webpack, Gulp, Vite, ESBuild...) ? Could we maybe see ?

Side note, I believe `require('autoprefixer')` should not be inside the "plugins" array but should be configured in your PostCSS config file.
